src/VaultSharp/VaultSharp.csproj

```diff
@@ -35,6 +35,12 @@
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Release\VaultSharp.XML</DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>VaultSharp.snk</AssemblyOriginatorKeyFile>
```

Comment:
You do not need to procure a certificate to strong name an assembly, whether it is distributed via NuGet or other means. Perhaps you are getting confused with creating a digital signature using `signtool.exe`.

Merely merging this pull request will not solve my team's issue. We are referencing VaultSharp from a strong named assembly, meaning VaultSharp must be strong named. That is why I also strong named the VaultSharp test assembly. We would prefer to manage references via NuGet to take advantage of automatic upgrades. Many open source projects distribute strong named versions of assemblies, as mentioned above.

Please let me know if anything is unclear.